src/driver/src/DefaultDriver.cpp

```diff
@@ -89,7 +197,31 @@ genny::driver::DefaultDriver::OutcomeCode doRunLogic(
     auto actorSetup = metrics.operation("Genny", "Setup", 0u);
     auto setupCtx = actorSetup.start();
 
-    auto yaml = loadConfig(options.workloadSource, options.workloadSourceType);
+    auto config = loadConfig(options.workloadSource, options.workloadSourceType);
+    fs::path phaseConfigSource;
+    if (options.workloadSourceType == DefaultDriver::ProgramOptions::YamlSource::kString) {
+        phaseConfigSource = fs::current_path();
+    } else {
+        /*
+         * The directory structure for workloads and phase configs is as follows:
+         * /etc (or /src if building without installing)
+         *     /workloads
+         *         /[name-of-workload-theme]
+         *             /[my-workload.yml]
+         *     /phase_configs
+         *         /[name-of-actor]
+         *             /[name-of-phase-config-snippet.yml]
+         *
+         * The path to the phase config snippet is obtained as a relative path from the workload
+         * file.
+         */
+        phaseConfigSource =
+            fs::path(options.workloadSource).parent_path().parent_path().parent_path() /
+            "phase_configs";
```

Comment:
1. Not sure what to suggest but relying on such a hard-coded path may come back to haunt us.
2. We've used PascalCase so instead lets have the subdir be called `PhaseConfigs` (or maybe `phases` if multi-word-mixed-casing-conventions is too unpleasant)

Reply:
1. I wanted to have the path be set in an environment variable but people didn't seem to like this at the 6week checkin meeting. Which would you prefer between env var and hard code?
2. So we've been using snake case for directories like `third_party` and `value_party`. I just want to confirm that you want to change the casing. (I think `PhaseConfigs` is preferable to `phases` because it's more explicit)